ivy/functional/frontends/paddle/stat.py

```diff
@@ -50,6 +50,27 @@ def numel(x, name=None):
     return ivy.array(prod if prod > 0 else ivy.array(length, dtype=ivy.int64))
 
 
+@with_supported_dtypes({"float32", "float64"})
```

Comment:
You will also need to add a test for you function. Please refer to our [frontend testing documentation](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html) about this.